### DISCUSSION TITLE
Get a json response right away

### DISCUSSION THREAD

**Discussion by array05** [norkunas/youtube-dl-php#178] [Q&A] [answered]
Is it possible to remove the video download, and immediately get a json response that is saved to a folder, without saving the video file?

**Comment by norkunas** [comment, marked as answer]
Yes, you can use `Options::skipDownload(true)` for this case, it will return the collection of your videos metadata.
But if you also would like to access the original metadata files then set `Options::cleanupMetadata(false)`.

**Reply by array05** [reply]
**Thank you, you are the best**
